Px.Utils/Validation/SyntaxValidation/SyntaxValidationUtilityMethods.cs

```diff
@@ -384,6 +399,137 @@ internal static int GetLineChangesValidity(string value, PxFileSyntaxConf syntax
             return -1;
         }
 
+        private static bool TryGetTimeValueFormat(string input, out ValueType? valueFormat, PxFileSyntaxConf syntaxConf)
+        {
+            // Value has to start with the time interval indicator (TLIST by default)
+            if (!input.StartsWith(syntaxConf.Tokens.Time.TimeIntervalIndicator))
```

Comment:
Maybe use local variables for the configuration items that are long and/or used multiple times to keep the line lengths in check.